app/assets/javascripts/3dbio_viewer/src/data/repositories/EbiDbModelRepository.ts

```diff
@@ -47,6 +54,10 @@ export class EbiDbModelRepository implements DbModelRepository {
                 .value()
         );
     }
+    upload(options: UploadOptions): FutureData<unknown> {
+        const uploadResults = request(config.upload.searchUrl, options);
```

Comment:
We'll need a multipart/form-data POST to actually post these files, but let's leave this for the next PR, as here we concern ourselves mainly with the UI.